client/src/main/scala/drt/client/components/FlightsWithSplitsTable.scala

```diff
@@ -24,26 +24,34 @@ object FlightsWithSplitsTable {
 
   type BestPaxForArrivalF = Arrival => Int
 
+  val acl_arrival_status = ArrivalStatus("ACL Forecast")
+
+  val passengerFlights: Arrival => Boolean = apiFlight =>
+      (apiFlight.Status != acl_arrival_status) ||
+      (apiFlight.Status == acl_arrival_status && apiFlight.ServiceType.isEmpty) ||
+      (apiFlight.Status == acl_arrival_status && apiFlight.LoadFactor.exists(_ != 0) && apiFlight.ServiceType.exists(s => List("J", "S", "Q", "G", "B", "R", "C", "L") contains s) && apiFlight.MaxPax.exists(_ != 0))
+
   case class Props(flightsWithSplits: List[ApiFlightWithSplits],
                    queueOrder: Seq[Queue], hasEstChox: Boolean,
                    arrivalSources: Option[(UniqueArrival, Pot[List[Option[FeedSourceArrival]]])],
                    hasArrivalSourcesAccess: Boolean,
                    viewMode: ViewMode,
                    pcpPaxFn: Arrival => Int,
-                   hasTransfer: Boolean
+                   hasTransfer: Boolean,
+                   filterPassengerFlights: Boolean
                   )
 
   implicit val propsReuse: Reusability[Props] = Reusability.by((props: Props) => {
     (props.flightsWithSplits, props.arrivalSources).hashCode()
   })
 
+
   def ArrivalsTable(timelineComponent: Option[Arrival => VdomNode] = None,
                     originMapper: PortCode => VdomNode = portCode => portCode.toString,
-                    splitsGraphComponent: SplitsGraphComponentFn = (_: SplitsGraph.Props) => <.div()
+                    splitsGraphComponent: SplitsGraphComponentFn = (_: SplitsGraph.Props) => <.div(),
                    ): Component[Props, Unit, Unit, CtorType.Props] = ScalaComponent.builder[Props](displayName = "ArrivalsTable")
     .render_P(props => {
-
-      val flightsWithSplits = props.flightsWithSplits
+      val flightsWithSplits = if (props.filterPassengerFlights) props.flightsWithSplits.filter(fs => passengerFlights(fs.apiFlight)) else props.flightsWithSplits
```

Comment:
this if would be easier to read if you put in some new lines. We should re-look at adding some editor config to enforce this kind of thing.

Reply:
I can see other places in the class , where IF is in single line when it is simple small change with if . I am fine either way as I don't it makes it more readable but for consistency we could add scala style which help .

Reply:
I think the convention we try to follow is to wrap if a line is longer than 120, I think this file is pretty bad at not following that, it's not a big deal though.

Reply:
Cool , Definitely scala style code would be handy here